alembic/versions/2158aff077f4_upd_temperature.py

```diff
@@ -0,0 +1,39 @@
+"""upd temperature
+
+Revision ID: 2158aff077f4
+Revises: f92e90328dfd
+Create Date: 2024-04-21 00:21:58.389183
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '2158aff077f4'
+down_revision: Union[str, None] = 'f92e90328dfd'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('temperatures',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('city_id', sa.Integer(), nullable=False),
+    sa.Column('date_time', sa.DateTime(), nullable=False),
+    sa.Column('temperature', sa.Float(), nullable=False),
+    sa.ForeignKeyConstraint(['city_id'], ['cities.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_index(op.f('ix_temperatures_id'), 'temperatures', ['id'], unique=False)
```

Comment:
The index `ix_temperatures_id` is created on the `id` column. Since `id` is already a primary key, consider whether this additional index is necessary, as primary keys are indexed by default.